src/i18n/en.json

```diff
@@ -250,9 +250,18 @@
       "CurrencyConversionTipOver1": "One {name} is worth {value} Lunars.",
       "CurrencyConversionTipLunar": "Lunars are the standard basis of currency.",
       "CurrencyConversionTipUnder1": "One Lunar is worth {value} {name}s."
+    },
+    "confirmItemDeleteDialog": {
```

Comment:
Maybe this should be called `confirmCultDeleteDialog` since `confirmItemDeleteDialog` could be used for a more generic delete dialog.

Reply:
If you look in rqgActorSheet.confirmItemDelete you'll see that it actually IS the generic item delete, but it uses the "contentCult" for the content if it's a cult and otherwise just uses "content".   So I think "confirmItemDeleteDialog" makes sense.